ts/src/base/Exchange.ts

```diff
@@ -2800,6 +2798,55 @@ export default class Exchange {
         return this.accounts;
     }
 
+    buildOHLCVC (trades: Trade[], timeframe: string = '1m', since: number = 0, limit: number = 2147483647): OHLCVC[] {
```

Comment:
@kroitor can we add here an additional arg named `includeCount = true` so users could set it to false if they don't want to get OHLCV'C' format, instead remove last 'C' and return standard OHLCV format.

Reply:
I also thought about adding separate `buildOHLCV` method which just leaves out the count field, but I didn't feel ideal when introducing two methods that might arise more questions.